ctriface/manual_cleanup_test.go

```diff
@@ -91,7 +91,7 @@ func TestSnapLoad(t *testing.T) {
 
 	vmID = "2"
 
-	_, _, err = orch.LoadSnapshot(ctx, vmID, snap)
+	_, _, err = orch.LoadSnapshot(ctx, "1", vmID, snap)
```

Comment:
please replace contstants with a properly names variable for readability. In this case, can use `originalVmID`